packages/edit-widgets/src/store/controls.js

```diff
@@ -74,7 +75,7 @@ export function getWidgetToClientIdMapping() {
 export function getNavigationPostForMenu( menuId ) {
 	return {
 		type: 'SELECT',
-		registryName: 'core/edit-navigation',
```

Comment:
Let's leave it as a hardcoded string and revert the change in this line. We shouldn't expose new constants from `@wordpress/edit-navigation`. It is probably something complex that we better ignore in this PR.

@adamziel and @noisysocks, aren't `edit-widgets` and `edit-navigation` two packages that target two different pages? I'm curious how it works since `@wordpress/edit-widgets` doesn't depend on `@wordpress/edit-navigation` as you can learn from the `package.json` file changes proposed in this PR.

Reply:
It's probably just a mistake. `edit-widgets`'s store is originally copied from `edit-navigation`, seems like we just forgot to delete this.